drafts/concise_guide.ipynb

```diff
@@ -92,7 +92,66 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "When creating a response, FastHTML will automatically render tags using the to_xml function. "
+    "This example demonstrates key aspects of how FastHTML components handle attributes:\n",
+    "\n",
+    "```python\n",
+    "Label(\n",
+    "    \"Choose an option\", \n",
+    "    Select(\n",
+    "        Option(\"one\", value=\"1\", selected=True),  # Boolean True renders just the attribute name\n",
```

Comment:
If you used `value=2` here, you could remove the next line and merge its comment into this one. i.e. Show as many things per line as possible!